libminifi/include/core/Repository.h

```diff
@@ -56,17 +56,19 @@ namespace core {
 
 #define REPOSITORY_DIRECTORY "./repo"
 #define MAX_REPOSITORY_STORAGE_SIZE (10*1024*1024)  // 10M
-#define MAX_REPOSITORY_ENTRY_LIFE_TIME (600000)  // 10 minute
-#define REPOSITORY_PURGE_PERIOD (2500)  // 2500 msec
+constexpr std::chrono::milliseconds MAX_REPOSITORY_ENTRY_LIFE_TIME = std::chrono::minutes(10);
+constexpr std::chrono::milliseconds REPOSITORY_PURGE_PERIOD = std::chrono::milliseconds(2500);
```

Comment:
```suggestion
using namespace std::literals;
constexpr auto MAX_REPOSITORY_ENTRY_LIFE_TIME = 10m;
constexpr auto REPOSITORY_PURGE_PERIOD = 2500ms;
```

Reply:
good idea, changed them to auto in https://github.com/apache/nifi-minifi-cpp/pull/1225/commits/78f8658fde802c73a5965dd734a544acbc37dca3, I've left out the using namespace std::literals; because this is a header